### ISSUE TITLE
Removal of elem_type input attribute support for PCCC prevents users from reading it out

### ISSUE THREAD

**Issue by kyle-github** [libplctag/libplctag#393]
There is a use case that is not covered here.   Users that want the type of a PCCC tag used to be able to read out the integer attribute "elem_type" to get the data file type.   That got dropped when the data file parsing code got better a while back.   

This is a regression.

**Comment by kyle-github** [comment]
Here is the list of values for PCCC:

```c
               PCCC_FILE_UNKNOWN        = 0x00, /* UNKNOWN! */
               PCCC_FILE_ASCII          = 0x8e,
               PCCC_FILE_BCD            = 0x8f,
               PCCC_FILE_BIT            = 0x85,
               PCCC_FILE_BLOCK_TRANSFER = 0x00, /* UNKNOWN! */
               PCCC_FILE_CONTROL        = 0x88,
               PCCC_FILE_COUNTER        = 0x87,
               PCCC_FILE_FLOAT          = 0x8a,
               PCCC_FILE_INPUT          = 0x8c,
               PCCC_FILE_INT            = 0x89,
               PCCC_FILE_LONG_INT       = 0x91,
               PCCC_FILE_MESSAGE        = 0x92,
               PCCC_FILE_OUTPUT         = 0x8b,
               PCCC_FILE_PID            = 0x93,
               PCCC_FILE_SFC            = 0x00, /* UNKNOWN! */
               PCCC_FILE_STATUS         = 0x84,
               PCCC_FILE_STRING         = 0x8d,
               PCCC_FILE_TIMER          = 0x86
```

**Comment by kyle-github** [comment]
Fixed in v2.5.3.
